plugin/metrics/prometheus/metricsstore/reader_test.go

```diff
@@ -387,12 +380,11 @@ func TestGetErrorRates(t *testing.T) {
 			wantName:         "service_operation_error_rate",
 			wantDescription:  "error rate, computed as a fraction of errors/sec over calls/sec, grouped by service & operation",
 			wantLabels: map[string]string{
-				"operation":    "/OrderResult",
```

Comment:
why is this removed?

Reply:
This is a valid question @varshith257 . 

These two lines probably need updating: https://github.com/jaegertracing/jaeger/blob/22851437b082df712b2bc7e3da0e13ed222bf944/plugin/metrics/prometheus/metricsstore/reader_test.go#L903-L904